macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -325,11 +325,31 @@ open class AvatarView: NSView {
 	///
 	/// - note: Internal visibility exists only for unit testing
 	@objc(getColorForIndex:)
+	@available(*, deprecated, message: "Use getInitialsColorSetFromPrimaryText:secondaryText: instead")
 	public static func getColor(for index: Int) -> ColorSet {
 		let avatarBackgroundColors = AvatarView.avatarColors
 		return avatarBackgroundColors[index % avatarBackgroundColors.count]
 	}
 
+	/// Returns a color table index for a given display name
+	/// - parameter identifyingString: the unique identifying string for the user
+	///
+	/// - Returns: a unique index generated by walking backwards through the letters in the \p identifyingString
+	/// creating a cumulative hash
+	///
+	/// - note: the returned index is not limited to any size and should not be considered safe for indexing into an array without
+	/// first checking the size of the array in question
+	@objc(colorIndexForIdentifyingString:)
+	@available(*, deprecated, message: "Use getInitialsColorSetFromPrimaryText:secondaryText: instead")
+	public static func colorIndex(for identifyingString: String) -> Int {
```

Comment:
This is being put back for the Outlook scenario only? Or all devmain apps will still use it?

Reply:
outlook scenario only

Reply:
Based on your code review open in devmain, it seems like all the current uses in devmain will still use this, no?

Reply:
yes, after we merge this PR these methods will still be required for outlook but rest of cocoaui will use the new API's